zulip_bots/zulip_bots/bots/beeminder/fixtures/test_invalid.json

```diff
@@ -0,0 +1,20 @@
+{
+    "request": {
+        "api_url": "https://www.beeminder.com/api/v1/users/aaron/goals/goal/datapoints.json",
+        "method": "POST",
+        "params": {
+            "auth_token": "someInvalidKey",
+            "value": "5"
```

Comment:
this should be `test` instead of `5`

Reply:
I've changed `test` to `5` in `beeminder.py` to be on safer side because passing `test` in function can cause 422 instead of 401 :)